src/lua/elrsV2.lua

```diff
@@ -967,20 +1037,24 @@ local function init()
 end
 
 -- Main
-local function run(event)
+local function run(event, touchState)
```

Comment:
Is this backward compatible i.e. will it still work on radios without touch?

Reply:
Tested with Zorro running EdgeTX v2.8.0-dev, the LUA runs as well.